code/game/turfs/open/floor/light_floor.dm

```diff
@@ -70,18 +70,17 @@
 	set_light(0)
 	return ..()
 
-/turf/open/floor/light/attackby(obj/item/W, mob/user, params)
-	if(W.tool_behaviour == TOOL_MULTITOOL)
-		. = ..()
-		if(.)
-			return
-		if(!can_modify_colour)
-			return
-		var/choice = show_radial_menu(user,src, lighttile_designs, custom_check = FALSE, radius = 36, require_near = TRUE)
-		if(!choice)
-			return FALSE
-		currentcolor = choice
-		update_icon()
+/turf/open/floor/light/multitool_act(mob/living/user, obj/item/I)
+	. = ..()
+	if(.)
+		return
+	if(!can_modify_colour)
+		return
+	var/choice = show_radial_menu(user,src, lighttile_designs, custom_check = FALSE, radius = 36, require_near = TRUE)
+	if(!choice)
+		return FALSE
+	currentcolor = choice
+	update_icon()
 
```

Comment:
```suggestion
	if(!choice)
		return
	currentcolor = choice
	update_icon()
	return TRUE
```
If you don't return `TRUE` at the end of this proc it will continue with the attack.